### ISSUE TITLE
Support magic number assignment on objects outside of method definitions 

### ISSUE THREAD

**Issue by Bodacious** [Bodacious/no-magic-numbers-cop#2]
## What 

Do not report a violation when assigning magic numbers on objects outside of a method.

``` ruby 
# No offence
@foo.bar = 1

# No offence
foo.bar = 1

# Offence
def method_name
  @foo.bar = 1
end
```

## Why

This pattern is typically used to configure a library. The setter methods are usually descriptive enough that we don't need to consider them "magic numbers"
